.gitignore

```diff
@@ -29,15 +37,13 @@ Cargo.lock
 
 # Temporary files
 *.000
-*.bak
```

Comment:
Why remove these three?

Reply:
IDE warned about duplicate entries so I removed the less relevant duplicates.